packages/protocol/test/RequesterStore.sol.js

```diff
@@ -0,0 +1,49 @@
+/* globals context ethers */
+
+const { expect } = require('chai');
+
+let airnodeRrp;
+let roles;
+
+beforeEach(async () => {
+  const accounts = await ethers.getSigners();
+  roles = {
+    deployer: accounts[0],
+    requesterAdmin: accounts[1],
+    client: accounts[2],
+    randomPerson: accounts[9],
+  };
+  const airnodeRrpFactory = await ethers.getContractFactory('AirnodeRrp', roles.deployer);
```

Comment:
Wouldn't it be better to just deploy RequesterStore, instead of the whole AirnodeRrp?

Reply:
> Wouldn't it be better to just deploy RequesterStore, instead of the whole AirnodeRrp?

hmmm, maybe? 🤔  If yes then this would also apply to all other contracts except fro AirnodeRrp.

The thing is that my understanding is that only AirnodeRrp is deployable and other parent contracts should not be deployed on their own. It's just a way to split up the code to make the contract more readable and maintainable.

@bbenligiray any thoughts on this?

Reply:
Actually, yes thats what I wanted to point out for TemplateStore, Convenience etc as well. My thinking is that shouldn't you be able to test these contracts on their own and if there is a functionality to test that requires inheriting this contract then that should be done separately. In the case of RequesterStore no extra functionality needs to be tested if AirnodeRrp is deployed.